spec/controllers/peer_reviews_controller_spec.rb

```diff
@@ -263,6 +263,87 @@
           expect(PeerReview.review_exists_between?(@reviewer, @reviewee)).to be false
         end
       end
+
+      context 'selected reviewer has existing peer review data' do
+        before do
+          @reviewer = Grouping.find_by(id: @selected_reviewer_group_ids[0])
+          @reviewee = Grouping.find_by(id: @selected_reviewee_group_ids[1])
+          allow_any_instance_of(PeerReview).to receive(:has_marks_or_annotations?).and_return(true)
+          selected_group = {}
```

Comment:
It's not really necessary to create separate hash variables like this, you can define it inline as `{ @reviewee.id => { @reviewer.id => true } }` or something similar